boards/arm/mimxrt685_evk/pinmux.c

```diff
@@ -13,7 +13,7 @@ static int mimxrt685_evk_pinmux_init(const struct device *dev)
 
 #if DT_NODE_HAS_COMPAT_STATUS(DT_NODELABEL(flexcomm0), nxp_lpc_usart, okay) && CONFIG_SERIAL
 	/* USART0 RX,  TX */
-	const uint32_t port0_pin1_config = (/* Pin is configured as FC0_TXD_SCL_MISO_WS */
+	uint32_t port0_pin1_config = (/* Pin is configured as FC0_TXD_SCL_MISO_WS */
```

Comment:
Please move the const changes to a separate commit, they appear unrelated to I2S

Reply:
Sure I will move to a separate commit.